fermenter-mda/src/main/java/org/technologybrewery/fermenter/mda/notification/NotificationService.java

```diff
@@ -109,19 +121,21 @@ public void recordNotifications(MavenProject project) {
             Map<String, Notification> notificationsForFile = entry.getValue();
             int i = 0;
             for (Map.Entry<String, Notification> subMapEntry : notificationsForFile.entrySet()) {
-                File notificationParentFile = getNotificationParentFile(projectParentFile, subMapEntry.getValue());
-                File outputFile = new File(notificationParentFile, FilenameUtils.getName(fileName + "-" + UUID.randomUUID().toString() + "." + JSON));
-                try {
-                    Notification notification = subMapEntry.getValue();
-                    FileUtils.forceMkdir(notificationParentFile);
-                    FileUtils.write(outputFile, getNotificationJson(notification), Charset.defaultCharset());
-                    if (notification instanceof VelocityNotification) {
-                        VelocityNotification velocityNotification = (VelocityNotification) notification;
-                        velocityNotification.writeExternalVelocityContextProperties(project.getBasedir());
+                if(suppressedMessages == null || (suppressedMessages != null && !suppressedMessages.contains(subMapEntry.getValue().getKey()))) {
```

Comment:
A:
```suggestion
                if(suppressedMessages == null || !suppressedMessages.contains(subMapEntry.getValue().getKey())) {
```
I'd also pull the assignment `Notification notification = subMapEntry.getValue()` up above the if statement to make the check more readable.